app/api/dao/user_extension.py

```diff
@@ -73,3 +72,41 @@ def get_user_additional_data_info(user_id):
                 "mobile": result.additional_info["mobile"],
                 "personal_website": result.additional_info["personal_website"]
             }
+        return
+
+    @staticmethod
+    def update_user_additional_info(data):
+        """Updates a user_extension instance.
+
+        Arguments:
+            data: A list containing user's id, boolean value of whether or not
+            the user is representing an organization, as well as their timezone
+    
+        Returns:
+                A dictionary containing "message" which indicates whether or not the user_exension was created successfully and "code" for the HTTP response code.
+        """
+
+        user_id = int(AUTH_COOKIE["user_id"].value)
+        if user_id == 0:
```

Comment:
If not user_id

Reply:
Yep, it's now removed completely as I'm applying the EAFP `try catch` 😁
<img width="694" alt="Screen Shot 2020-07-29 at 5 32 21 pm" src="https://user-images.githubusercontent.com/29667122/88770861-c1e31700-d1c1-11ea-9bd9-6cbcdacfa911.png">

Reply:
Yep, I removed it completely as now I'm applying EAFP `try catch` instead 😁

<img width="694" alt="Screen Shot 2020-07-29 at 5 32 21 pm" src="https://user-images.githubusercontent.com/29667122/88771198-3c139b80-d1c2-11ea-87a9-00c74db43a70.png">

Reply:
Very happy to see more usage of EAFP!